src/style/index.scss

```diff
@@ -5,6 +5,8 @@ html, body {
   width: 100%;
   padding: 0;
   margin: 0;
+  font: 14px/1.3 Roboto,'Helvetica Neue',arial,helvetica,sans-serif;
```

Comment:
Let's move these changes into their own rule and put the usual disclaimer there.

Reply:
could just assume all of the styling is temporary, right? we don't have a completed design.

Reply:
That's fair for this resource, but some styles (such as image-rendering) are intentional.

Reply:
added